dapp-oeth/src/components/wrap/WrapHomepage.js

```diff
@@ -208,24 +209,33 @@ const WrapHomepage = ({
       setInputAmount('')
 
       await rpcProvider.waitForTransaction(result.hash)
-
-      analytics.track('Wrap succeeded', {
-        category: 'wrap',
-        label: metadata.coinUsed,
-        value: metadata.swapAmount,
+      event({
+        'event': 'wrap_complete',
+        'wrap_type': swapMode,
+        'wrap_token': wrapTokenUsed,
+        'wrap_amount': wrapTokenAmount,
+        'wrap_address': '',
+        'wrap_tx': ''
```

Comment:
Should this be `result.hash` rather than being a empty string?

Reply:
Will fix forward, just wanted to get this up, address/tx are known empty for now